frontend/package.json

```diff
@@ -10,7 +10,7 @@
     "@types/node": "^16.18.11",
     "@types/react": "^18.0.27",
     "@types/react-dom": "^18.0.10",
-    "axios": "^1.2.5",
+    "axios": "^1.3.2",
```

Comment:
your frontend package.lock diff looks particularly large for such a small change - can you try deleting the package-lock and re-generating it? (just `npm i`)

Reply:
Fixed!